Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -417,7 +428,6 @@ RefPtr<StyleRuleBase> CSSParserImpl::consumeQualifiedRule(CSSParserTokenRange& r
     if (allowedRules == KeyframeRules)
         return consumeKeyframeStyleRule(prelude, block);
 
-    ASSERT_NOT_REACHED();
```

Comment:
Ditto.

Reply:
Same explanation (cf early function call + return in the previous lines).